includes/admin/feedzy-rss-feeds-admin.php

```diff
@@ -851,4 +851,29 @@ public function ajax() {
 	public function feedzy_remove_elementor_feature( $manager_object, $experimental_data ) {
 		$manager_object->remove_feature( 'e_hidden_wordpress_widgets' );
 	}
+
+	/**
+	 * Remove legacy widget.
+	 *
+	 * @param array $list Black list widgets.
+	 * @return array
+	 */
+	public function feedzy_remove_elementor_widgets( $list ) {
+		global $post;
+
+		$black_list = array( 'feedzy_wp_widget' );
+		$data       = \Elementor\Plugin::$instance->documents->get( $post->ID )->get_elements_data();
```

Comment:
@girishpanchal30 can check method existance and or add a check to the elementor version, to make sure it does not throw fatal error on older versions.